udata_front/views/reuse.py

```diff
@@ -1,8 +1,8 @@
-from flask import abort, request, url_for
-from werkzeug.contrib.atom import AtomFeed
+from flask import abort, request, url_for, make_response
+from feedgenerator.django.utils.feedgenerator import Atom1Feed
```

Comment:
AtomFeed is not longer supported by Werkzeug. Changed to feedgenerator, inspired from https://github.com/lektor/lektor-atom/commit/1d143031fb9359e246a1fdb73962ec2c1082f632. See also https://github.com/ckan/ckan/pull/5054. The best lib is not an easy choice